WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -174,11 +174,18 @@ targetted_decisions = {
 				}
 			}
 			controls_religion = no
+<<<<<<< HEAD
+			NOT = { 
+				trait = zealous
+				holy_order = yes
+			}
+=======
 			NOT = { trait = zealous }
 			any_spouse = {
 				NOT = { religion = ROOT }
 			}
 
+>>>>>>> refs/remotes/loup99/master
```

Comment:
I don't understand what that means, but I will discard this as soon as I can fix my desktop PC.